src/lib/stack.js

```diff
@@ -65,5 +65,7 @@ export default {
   get,
   updateAccessToken: (...args) => current().updateAccessToken(...args),
   logout: (...args) => current().logout(...args),
-  cozyFetchJSON: (...args) => current().cozyFetchJSON(...args)
+  cozyFetchJSON: (...args) => current().cozyFetchJSON(...args),
+  // allow to connect other getters
```

Comment:
Allows

I have trouble understanding the comment, it could be more detailed IMHO.